### DISCUSSION TITLE
Kubernetes deployment failing

### DISCUSSION THREAD

**Discussion by andreasteich** [plausible/analytics#1331] [Self-Hosted Support]
After following the official documentation on how to deploy `plausible` within kubernetes, it wouldn't work at the moment. The deployment works, but creating the `plausible pod` fails when executing the `plausible-init` container. What's even more tricky, running `kubectl logs pod-id -c plausible-init` doesn't provide any log data. 

Any suggestions what went wrong?

In general, the following steps were made:
- generate `SECRET_KEY_BASE` according to documentation
- adjusting the `plausible-conf.env` file
- `kubectl -n plausible create secret generic plausible-db-user --from-literal='username=postgres' --from-literal='password=new_test_password_2021'`
- `kubectl -n plausible create secret generic plausible-events-db-user --from-literal='username=clickhouse' --from-literal='password=new_test_password_2021'`
- applied the kubernetes deployment

**Comment by myusuf3** [comment]
you figure this out?

**Reply by myusuf3** [reply]
issue was my secret wasn't long enough.
